src/components/pages/Login.vue

```diff
@@ -12,6 +12,7 @@
               <input
                 class="input is-medium email"
                 type="text"
+                v-model="email"
```

Comment:
Always put  `v-model` at the bottom for consistency reasons.

Reply:
It's fixed